content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._en.md

```diff
@@ -1,36 +1,119 @@
 ---
 layout: help
-title: How to add images of your state-issued ID
+title: How to add images of your driver’s license or state ID card
 category: verify-your-identity
 permalink: /help/verify-your-identity/how-to-add-images-of-your-state-issued-id/
 order: 3
 redirect_from:
   - /en/help/verify-your-identity/how-to-add-images-of-your-state-issued-id/
+do_list: 
+  - Do use a high resolution camera like a smartphone or tablet camera. Your computer webcam may not take clear photos.
+  - Do use a solid, dark background
+  - Do take your photos in a well-lit area with indirect light
+  - Do check that ID barcodes are not damaged or dirty
+  - Do try to hold your phone still while taking the photos. It may help to prop your arms on the table to steady yourself.
+  
+alert: <strong>If you do not have a <a href="/help/verify-your-identity/accepted-state-issued-identification/">valid drivers' license or state ID card</a>, you cannot use Login.gov for identity verification.</strong> Please contact the partner agency’s help center to find out what you can do instead."
```

Comment:
There must be a better way to do this link? I can't get the {{ }} syntax to work here though.

Reply:
We could avoid using the page frontmatter and do this in the content of the template itself.

Here's a good example to follow, which is very similar in creating content including `{{ }}` and using it for the alert partial:

https://github.com/18F/identity-site/blob/9026dc67b54095a2a7861c814f3f28e128c670bb/_layouts/contact_us.html#L28-L34